tools/peptideshaker/searchgui.xml

```diff
@@ -168,6 +171,27 @@
                 -myrimatch_max_peak ${myrimatch.myrimatch_max_peak}                    
             #end if
 
+            #*
+            #if $andromeda.andromeda_advanced == "yes"
+                -andromeda_max_pep_mass ${andromeda.andromeda_max_pep_mass}
```

Comment:
I'll remove it.   We only need in the command line generation:  -andromeda 0
Otherwise, the default is 1 (run andromeda).